<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineDistortionEstimatorParametersControlWidget</class>
 <widget class="QWidget" name="LineDistortionEstimatorParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LineDistortionEstimatorParameters</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Cost Algorithm</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="costAlgorithmComboBox">
       <property name="currentIndex">
         <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Line Deviation Cost</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Joint Angle Cost</string>
        </property>
       </item>
       <property name="toolTip">
         <string>Cost Algorithm</string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Iteration Number</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="iterationNumberSpinBox">
       <property name="toolTip">
         <string>Iteration Number</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Polinom Degree</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="polinomDegreeSpinBox">
       <property name="toolTip">
         <string>Polinom Degree</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>Simple Jacobian</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="simpleJacobianCheckBox">
       <property name="toolTip">
         <string>Simple Jacobian</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>Even powers only</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="evenPowersOnlyCheckBox">
       <property name="toolTip">
         <string>Even powers only</string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>Estimate Tangent</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="estimateTangentCheckBox">
       <property name="toolTip">
         <string>Estimate Tangent</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>Estimate Center</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="estimateCenterCheckBox">
       <property name="toolTip">
         <string>Estimate Center</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>costAlgorithmComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>iterationNumberSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>polinomDegreeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>simpleJacobianCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>evenPowersOnlyCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>estimateTangentCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>estimateCenterCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LineDistortionEstimatorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
